src/Illuminate/Foundation/Bootstrap/LoadConfiguration.php

```diff
@@ -79,6 +79,13 @@ protected function getConfigurationFiles(Application $app)
             $files[$nesting.basename($file->getRealPath(), '.php')] = $file->getRealPath();
```

Comment:
Btw, here we could do:

    $files[$nesting.$file->getBasename('.php')] = $file->getRealPath();